bot/build.sh

```diff
@@ -287,6 +287,22 @@ if [[ ! -z ${SHARED_FS_PATH} ]]; then
     BUILD_STEP_ARGS+=("--host-injections" "${SHARED_FS_PATH}/host-injections")
 fi
 
+# opt-in to use unionfs overlay tool and debian12 container for EESSI version 2025.06
+if [[ "${EESSI_VERSION_OVERRIDE} == "2025.06" ]]; then
+    BUILD_STEP_ARGS+=("--container" "docker://ghcr.io/eessi/build-node:debian12")
+    BUILD_STEP_ARGS+=("--overlay-tool" "unionfs")
+fi
+
+# allow specifying which container image to use via $EESSI_CONTAINER_IMAGE
+if [[ ! -z "${EESSI_CONTAINER_IMAGE}" ]]; then
```

Comment:
If both `$EESSI_VERSION == 2025.06` and `$EESSI_CONTAINER_IMAGE` are set, it will pass this argument twice? Which one will take precedence in that case?